github-for-jira.sd.yml

```diff
@@ -138,7 +140,7 @@ workers:
 
 environmentOverrides:
 #  Uncomment lines 140-149 if you want remote debugging in ddev
-#  ddev:
+  ddev:
```

Comment:
Home come this is needed?

Reply:
I thought Amanda just fixed it in a separate PR. Maybe you need to merge main

Reply:
It was commented by accident